src/item.js

```diff
@@ -21,7 +21,6 @@ const createItem = (id, texture, preview, pubSub) => {
     clonedSprite.y = sprite.y;
     clonedSprite.width = sprite.width;
     clonedSprite.height = sprite.height;
-    clonedSprite.angle = sprite.angle;
```

Comment:
Just curious, why do we not clone the angle anymore?

Reply:
Actually I don't remember why we clone the angle 😂If the angle is cloned, when temp-depile, the cloned sprite will also be rotated. But I think the cloned sprite doesn't need to be rotated?